### PR TITLE
change: use member tag & avatarURL instead of Isekai Music

### PR DESCRIPTION
**Pull request by iceisblue**

### CODE REVIEW

**Comment by Skillz4Killz** [comment]
If we do change it can we instead put the user in the author instead?

File: src/commands/voice/playing.ts [hunk 1]
```diff
@@ -17,8 +17,8 @@ createCommand({
 
     const embed = new Embed()
       .setAuthor(
-        "Isekai Music",
-        "https://cdn.discordapp.com/avatars/833417537449754657/097a32a4f7bb5821f5383c4ddeb69aa8.png?size=256",
+        `${message.guild?.bot?.tag}`,
```
Comment (Skillz4Killz) [inline src/commands/voice/playing.ts:20]:
```suggestion
        `${message.tag}`,
```

File: src/commands/voice/playing.ts [hunk 2]
```diff
@@ -17,8 +17,8 @@ createCommand({
...
+        message.guild?.bot?.avatarURL,
```
Comment (Skillz4Killz) [inline src/commands/voice/playing.ts:21]:
```suggestion
        message.member?.avatarURL,
```

File: src/commands/voice/queue.ts [hunk 3]
```diff
@@ -16,8 +16,8 @@ createCommand({
 
     const embed = new Embed()
       .setAuthor(
-        "Isekai Music",
-        "https://cdn.discordapp.com/avatars/833417537449754657/097a32a4f7bb5821f5383c4ddeb69aa8.png?size=256",
+        `${message.guild?.bot?.tag}`,
```
Comment (Skillz4Killz) [inline src/commands/voice/queue.ts:19]: same as above comments here

**Comment by iceisblue** [comment]
Should be good now.

**Pull request merged**